### PR TITLE
Mvs first commit

### PR DESCRIPTION
**Pull request by marcio-santos-zocdoc**
@Zocdoc/pinky 

Created python project and package. Install scripts to bash and ps1.
Need to change the TC build configs to separate selenium from the rest.

### CODE REVIEW

File: install.ps1 [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+Write '======================='
+$location = (Get-Item -Path ".\" ).FullName
+Push-Location
+Set-Location C:\
+pip install -e $location -I
+Pop-Location
+
+Write 'AVAILABLE COMMANDS'
+Write '=================='
+Write 'run-monolith-ci'
```
Comment (chris-smith-zocdoc) [inline install.ps1]: Is this just available on my path after I run the install script?

File: run_monolith_ci/run_monolith_ci.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+from git import Repo
+import requests
+import sys, getopt
+
+def trigger_run(build_config_name, branch_name) :
+    url = "https://teamcity.east.zocdoccloud.net/guestAuth/action.html?add2Queue={0}&branchName={1}".format(build_config_name, branch_name)
+    r = requests.post(url)
+    if r.status_code == 200 :
+        msg = "Triggered CI run with branch name = {}".format(branch_name)
+        print(msg)
```
Comment (chris-smith-zocdoc) [inline run_monolith_ci/run_monolith_ci.py]: can you print out the link to the build?

File: run_monolith_ci/run_monolith_ci.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+def trigger_run(build_config_name, branch_name) :
+    url = "https://teamcity.east.zocdoccloud.net/guestAuth/action.html?add2Queue={0}&branchName={1}".format(build_config_name, branch_name)
+    r = requests.post(url)
+    if r.status_code == 200 :
+        msg = "Triggered CI run with branch name = {}".format(branch_name)
+        print(msg)
+
+def print_options() :
+    print('usage: run-monolith-ci [-i,--include-selenium] [-b,--branch_name BRANCH_NAME]')
```
Comment (chris-smith-zocdoc) [inline run_monolith_ci/run_monolith_ci.py]: why doesn't this run selenium by default? are we going to disable **all** automatic branch runs by default? In the principals meeting I know we talked about potentially leaving everything except selenium on by default, and you would run selenium using this command

File: run_monolith_ci/run_monolith_ci.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+            branch_name = v
+    if not branch_name :
+        zd_path= 'C:\src\zocdoc.web\zocdoc_web'
+        branch_name = Repo(zd_path).active_branch.name
+    return branch_name, include_selenium
+
+def main(args=None) :
+
+    branch_name, include_selenium = parse_args(args)
+    trigger_run('Monolith_MarcioMonolithCi_TriggerTests', branch_name)
```
Comment (colin-hanson-zocdoc) [inline run_monolith_ci/run_monolith_ci.py]: Monolith_**Marcio**MonolithCi_TriggerTests

File: run_monolith_ci/run_monolith_ci.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        if k == '-h' :
+            print_options()
+            sys.exit()
+        elif k == '-i' or k == '--include-selenium' :
+            include_selenium = True
+        elif k == '-b' or b == '--branch_name' :
+            branch_name = v
+    if not branch_name :
+        zd_path= 'C:\src\zocdoc.web\zocdoc_web'
+        branch_name = Repo(zd_path).active_branch.name
```
Comment (colin-hanson-zocdoc) [inline run_monolith_ci/run_monolith_ci.py]: Gotta love python, what a handy module!

File: run_monolith_ci/run_monolith_ci.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        opts, args = getopt.getopt(args,"hib:",["include-selenium","branch_name="])
+    except getopt.GetoptError:
+        print_options()
+        sys.exit()
+
+    for k, v in args :
+        if k == '-h' :
+            print_options()
+            sys.exit()
+        elif k == '-i' or k == '--include-selenium' :
```
Comment (colin-hanson-zocdoc) [inline run_monolith_ci/run_monolith_ci.py]: I think we should probably go with -s as the shorthand for this

File: install.ps1 (hunk 1 above)
Comment (marcio-santos-zocdoc) [inline install.ps1]: yes. Maybe you have a better suggestion, @chris-smith-zocdoc ?

File: run_monolith_ci/run_monolith_ci.py (hunk 3 above)
Comment (marcio-santos-zocdoc) [inline run_monolith_ci/run_monolith_ci.py]: Yeah, I will do that after I split the build configs.

File: run_monolith_ci/run_monolith_ci.py (hunk 6 above)
Comment (marcio-santos-zocdoc) [inline run_monolith_ci/run_monolith_ci.py]: 👍

File: run_monolith_ci/run_monolith_ci.py (hunk 5 above)
Comment (marcio-santos-zocdoc) [inline run_monolith_ci/run_monolith_ci.py]: Python is nice. But sometimes slaps me in the face and says 'I'm not ruby!'

File: install.ps1 (hunk 1 above)
Comment (chris-smith-zocdoc) [inline install.ps1]: macrio explained this to me in person 👍

File: run_monolith_ci/run_monolith_ci.py (hunk 2 above)
Comment (marcio-santos-zocdoc) [inline run_monolith_ci/run_monolith_ci.py]: Not using the https api with guest auth :/

**Pull request merged**